stack/eslint-plugin-packlets/src/Path.ts

```diff
@@ -2,9 +2,113 @@
 // See LICENSE in the project root for license information.
 
 import * as path from 'path';
+import * as fs from 'fs';
+
+export type ParsedPath = path.ParsedPath;
 
-// These helpers are borrowed from @rushstack/node-core-library
 export class Path {
```

Comment:
Let's put together some tests for this.

Reply:
Since the original code worked fine on Linux and Windows, but failed on Mac, there isn't an easy way to write tests for that. We would need a way to deterministically simulate API behavior for different OS's. Or else our CI would need to do builds on all 3 OS's.

I can try to write a test for the string manipulations in `_relativeCaseInsensitive()`.

Reply:
Added some tests and fixed some obscure edge cases